metagraph/src/graph/alignment/aligner_chainer.cpp

```diff
@@ -1,63 +1,91 @@
 #include "aligner_chainer.hpp"
 
+#include <unordered_set>
```

Comment:
Why not `hopscotch_set`?

Reply:
I need it for `multiset`

Reply:
Ok, if it's not critical for the performance, it's fine.